PHP/PHPhomework4.php

```diff
@@ -0,0 +1,22 @@
+<?php
+//0～100
+$array = array();
+array_push($array,2);
+
+for ($i = 3;$i <= 100;$i++) {
```

Comment:
Please check coding standard of syntax `for` https://www.php-fig.org/psr/psr-2/ and fix it.